analyses/sample-distribution-analysis/.gitignore

```diff
@@ -1,2 +1,4 @@
 histology-pie_files
 histology-treemap_files
+# The nonessential output produced in the root directory of this repository when we run `analyses/sample-distribution-analysis/run-sample-distribution.sh`
```

Comment:
```suggestion
# The nonessential output produced in the root directory of this repository when we run `run-sample-distribution.sh`
```

Reply:
I think that since it's in the same directory, the path should only be the script name.

Reply:
Good idea -- added in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/79/commits/148f37617b15ab575980249f65ba18c020eb3b17